docs/website/blog/2023-11-27-dlt-data-lineage.md

```diff
@@ -0,0 +1,259 @@
+
+```python
+pip install -r requirements.txt
+```
+
+## Loading the data
+As a first step, we will load the sales data from skate shops online and physical store into BigQuery. In addition to the sales data, we will also ingest the dlt load_info into BigQuery. This will help us track changes in our pipeline.
+
+### Step 2: Adding the dlt pipeline code
+
+In the `data_lineage.py` file remove the default code and add the following:
+
+```python
+FILEPATH = "data/supermarket_sales.csv"
+FILEPATH_SHOPIFY = "data/orders_export_1.csv"
+
+class Data_Pipeline:
+    def __init__(self, pipeline_name, destination, dataset_name):
+        self.pipeline_name = pipeline_name
+        self.destination = destination
+        self.dataset_name = dataset_name
+
+    def run_pipeline(self, data, table_name, write_disposition):
+        # Configure the pipeline with your destination details
+        pipeline = dlt.pipeline(
+            pipeline_name=self.pipeline_name, destination=self.destination, dataset_name=self.dataset_name
+        )
+        # Run the pipeline with the provided data
+        load_info = pipeline.run(data, table_name=table_name, write_disposition=write_disposition)
+
+        # Pretty print the information on data that was loaded
+        print(load_info)
+        return load_info
+```
+
+Any changes in the underlying data are captured by the dlt load_info. To showcase this we will filter the data to remove the **Branch** and **Tags** columns from Store and Shopify data respectively and run the pipeline. Later, we will add back the columns and rerun the pipeline. These new columns added will be recorded in the load_info packages. 
+
+We will add the load_info back to BigQuery to use in our Dashboard. The Dashboard will provide an overview data lineage for our ingested data.
+
+```python
+if __name__ == "__main__":
+
+    data_store = pd.read_csv(FILEPATH)
+    data_shopify = pd.read_csv(FILEPATH_SHOPIFY)
+
+    #filtering some data. 
+    select_c_data_store = data_store.loc[:, data_store.columns.difference(['Branch'])]
+    select_c_data_shopify = data_shopify.loc[:, data_shopify.columns.difference(['Tags'])]
+
+    pipeline_store = Data_Pipeline(pipeline_name='pipeline_store', destination='bigquery', dataset_name='sales_store')
+    pipeline_shopify = Data_Pipeline(pipeline_name='pipeline_shopify', destination='bigquery', dataset_name='sales_shopify')
+
+		load_a = pipeline_store.run_pipeline(data=select_c_data_store, table_name='sales_info', write_disposition='replace')
+    load_b = pipeline_shopify.run_pipeline(data=select_c_data_shopify, table_name='sales_info', write_disposition='replace')
+
+    pipeline_store.run_pipeline(data=load_a.load_packages, table_name="load_info", write_disposition="append")
+    pipeline_shopify.run_pipeline(data=load_b.load_packages, table_name='load_info', write_disposition="append")
+```
+
+### Step 3: Run the dlt pipeline.
```

Comment:
titles should be in the same style, that is, if you put a dot `.` at the end of a sentence, then put it everywhere.